lib/form1010_ezr/service.rb

```diff
@@ -155,7 +159,7 @@ def configure_and_validate_form(parsed_form)
       post_fill_fields(parsed_form)
       validate_form(parsed_form)
       # Due to overriding the JSON form schema, we need to do so after the form has been validated
-      override_parsed_form(parsed_form)
+      HCA::OverridesParser.new(parsed_form).override
```

Comment:
This seemed a bit more logical than calling a method that calls this line of code.